gradle-hsql-plugin/gradlew

```diff
@@ -0,0 +1,164 @@
+#!/usr/bin/env bash
```

Comment:
does this sub-project need its own gradle wrapper?
Can it leverage the top level wrapper?